azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
```

Comment:
Can you please rename type field to name?

Reply:
Done.